src/config/constants.js

```diff
@@ -32,6 +33,14 @@ export const AUTHENTICATION_HOST =
   process.env.REACT_APP_AUTHENTICATION_HOST ||
   'http://localhost:3112';
 
+export const GRAASP_COMPOSE_HOST =
+  ENV_GRAASP_COMPOSE_HOST ||
+  process.env.REACT_APP_GRAASP_COMPOSE_HOST ||
+  'http://localhost:3111';
+
 // define a max height depending on the screen height
 // use a bit less of the height because of the header and some margin
 export const SCREEN_MAX_HEIGHT = window.innerHeight * 0.8;
```

Comment:
We might want to calculate this exactly. I.e. innerHeight minus header size.

Reply:
Actually the header size is never defined and depends on the content. And probably this screen size height is better defined depending on the height alone, as it let some margins to the elements. If you still want to depend on the header height, we can open an issue for that.